examples/Stratigraphic_model/serial.py

```diff
@@ -0,0 +1,68 @@
+from firedrake import *
+from firedrake.petsc import PETSc
+print = lambda x: PETSc.Sys.Print(x)
```

Comment:
`Print = PETSc.Sys.Print` will allow passing more that one argument (e.g. a comm), and will not override a keyword.